package.json

```diff
@@ -10,7 +10,7 @@
   "type": "module",
   "packageManager": "yarn@1.22.19",
   "devDependencies": {
-    "@endo/eslint-plugin": "^0.5.1",
+    "@endo/eslint-plugin": "^1.0.0",
```

Comment:
these 1.0s, will they all be semver? Are Endo maintainers committed to major very bump for all breaking changes? do deep imports count as the package API?

Reply:
Deep imports are expressly forbidden by an `"exports"` directive in every public Endo package except `captp`, `far`, `lockdown`, and `marshal`, and these must be fixed.

Endo has for some time committed to backward-compatibility for dependents using the default carat (`^`) range, which is more strict than semver.

Reply:
https://github.com/endojs/endo/pull/1907

Reply:
Tracking the broader issue for Agoric SDK https://github.com/Agoric/agoric-sdk/issues/8679